tests/calculators/test_qe.py

```diff
@@ -215,18 +228,18 @@ def test_qe_calculation(tmp_path, qe_cmd_and_pseudo):
     assert si2.arrays["QE_forces"][0] == approx(-1 * si2.arrays["QE_forces"][1])
 
 
-def test_wfl_Espresso_calc(tmp_path, qe_cmd_and_pseudo):
-    # command and pspot
-    qe_cmd, pspot = qe_cmd_and_pseudo
+def test_wfl_Espresso_calc(tmp_path, qe_profile_and_pseudo):
```

Comment:
Once the tests are shifted to rely on ase config and not on explicit profile construction, I would remove this test and only keep the`test_wfl_Espresso_calc_via_generic` below.

Reply:
See comment above